lidar3D:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  lidar_1_pris1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_pris1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  lidar_1_pris2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_pris2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  lidar_1_pris3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_pris3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  lidar_1_rev1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_rev1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  lidar_1_rev2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_rev2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  lidar_1_rev3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lidar_1_rev3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

